projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -306,11 +303,23 @@ jQuery( document ).ready( function ( $ ) {
 
 			var full_jp_logo_exists = $( '.jetpack-logo__masthead' ).length ? true : false;
 
-			$.get( window.jitm_config.api_root + 'jetpack/v4/jitm', {
+			// Original code
+			// const url = jitm_config.api_root + 'jetpack/v4/jitm';
+
+			// make an WP Ajax request to local site as REST API is not initialized on Simple sites.
+			const dotComURL = window.ajaxurl;
```

Comment:
Obviously this will need to be made conditional based on whether run on Simple or not.